core/micro-services/workflow-core/src/main/scala/edu/uci/ics/amber/exception/WorkflowRuntimeException.scala

```diff
@@ -0,0 +1,30 @@
+package edu.uci.ics.amber.exception
```

Comment:
Leave it in core package. No need for a sub exception package

Reply:
changed